Jenkinsfile

```diff
@@ -1009,7 +1010,8 @@ pipeline {
                     steps {
                         functionalTest inst_repos: el7_daos_repos(),
                                        inst_rpms: get_daos_packages('centos7') +
-                                                  ' ' + functional_rpms
+                                                  ' ' + functional_rpms +
+                                                  ' python2-tabulate '
```

Comment:
Ditto.

Reply:
> Do we need this since we run all cart tests in functional vm?

Without the `python2-tabulate` param, we get [missing `tabulate` error](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-3139/8/testReport/junit/(root)/TestError/Test___Functional___01___cart_rpc_cart_rpc_two_node_py_CartRpcTwoNodeTest_test_cart_rpc_defaultENV_sep_hosts_1_proto_np_1_a59c/).

Reply:
> > Do we need this since we run all cart tests in functional vm?
> 
> Without the `python2-tabulate` param, we get [missing `tabulate` error](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-3139/8/testReport/junit/(root)/TestError/Test___Functional___01___cart_rpc_cart_rpc_two_node_py_CartRpcTwoNodeTest_test_cart_rpc_defaultENV_sep_hosts_1_proto_np_1_a59c/).

I see that you are installing the tabulate package in every test stage, not just the Functional Stage where cart test is running. But is you are going for the rpm dependency route to daos-test rp, then I think won't apply anymore.

Reply:
60c8ef19a03e1a7e5aaee2a59e7c2ab60f6d9362